packages/components/modules/content-feed/web/NewContentPost/types.ts

```
@@ -0,0 +1,10 @@
+export interface ContentPostCreateForm {
+  content: string
+  images?: File[]
+}
+
+export type UploadableContentPostFiles = {
+  [key: `images.${number}`]: File | Blob
+}
+
+export interface INewContentPostProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)